packages/clerk-js/src/ui/icons/index.ts

```diff
@@ -21,6 +21,7 @@ export { default as Check } from './check.svg';
 export { default as CheckCircle } from './check-circle.svg';
 export { default as CheckmarkFilled } from './checkmark-filled.svg';
 export { default as ChevronDown } from './chevron-down.svg';
+export { default as ChevronUpDown } from './chevron-up-down.svg';
```

Comment:
Renamed `ArrowUpDown` to be `ChevronUpDown` to account for the newly added `ArrowsUpDown` icon.

No change in output:

![Screenshot 2025-05-05 at 10 08 13 PM](https://github.com/user-attachments/assets/d43603fe-418c-4d1a-aba7-4355ce469591)